docs(site): add usage guidelines WO1 (#1019)
ec-europa/europa-component-library: src/website/src/pages/ec/components/card/docs/usage.md

order: 1
---

import { Paragraph } from '@ecl/website-components';

<Paragraph size="lead">
  Cards act as an entry point to more detailed information. A card is container for a few short, related pieces of information. It roughly resembles a playing card in size and shape, and is intended as a linked, short representation of a conceptual unit.
</Paragraph>

## Anatomy

All elements are optional. Card layouts can vary to support the types of content they contain. The following elements are commonly found.

| Name        | Mandatory |                     Note |
| ----------- | :-------: | -----------------------: |
| Thumbnail   |           | SHOULD BE CLICKABLE !!!! |

> | Title       |           | SHOULD BE CLICKABLE !!!! |

Reviewer: I would say that the title is mandatory for a card...

| Description |           |                          |
| Meta        |           |                          |
| Tags        |           |                          |
| Infos       |           |                          |

## When to use

- For information browsing (as opposed to searching)
- To group heterogeneous types of content
- To provide a visual boundary, allowing users to easily navigate through
  - several distinct items

>   - To enhance information browsing

Reviewer: seems to miss a second bullet point here, following "through"

- The user goals that the card-based web design best responds to:
  - scrolling through
  - scanning through
  - browsing through

## When not to use

- For information searching (as opposed to browsing).
- When you have a small screen display and you want to offer immediate access to results it is better to avoid cards as they force users to scroll down and rely on their short-term memory. This creates a cognitive overload which harms UX.

## DO's

- Use cards to emphasize on contents
- Only display the most relevant information on each card
  illustration: card with a thumbnail, title, description
- Keep the amount of links on the cards as low as possible

## DON'Ts

- Use cards as an image gallery (link to image gallery)
- Use cards to display images or video (link to media container)

> - Avoid to displaying a large amount of cards

Reviewer: Phrasing is not consistent here (the two first bullets could start with "don't", the two last one with "do")

  illustration: card with multiple meta, tags

> - Avoid having too many links on a cards, keep it

Reviewer: missing the endof the line

order: 1
---

import { Link, Paragraph } from '@ecl/website-components';

<Paragraph size="lead">
  Cards act as an entry point to more detailed information. A card is container for a few short, related pieces of information. It roughly resembles a playing card in size and shape, and is intended as a linked, short representation of a conceptual unit.
</Paragraph>

## Anatomy

Card layouts can vary to support the types of content they contain.

| Name        | Mandatory |                     Note |
| ----------- | :-------: | -----------------------: |
| Thumbnail   |           | SHOULD BE CLICKABLE !!!! |
| Title       |     X     | SHOULD BE CLICKABLE !!!! |
| Description |           |                          |
| Meta        |           |                          |
| Tags        |           |                          |
| Infos       |           |                          |

## When to use

> - For information browsing (as opposed to searching)

Reviewer 2: ```suggestion
- Browsing for information (as opposed to searching)
```

> - To group heterogeneous types of content

Reviewer 2: ```suggestion
- Grouping heterogeneous types of content
```

> - To provide a visual boundary, allowing users to easily navigate through:

Reviewer 2: ```suggestion
- Allowing users to scan through content in small portions
```

>   - several distinct items

Reviewer 2: ```suggestion
  - To display structured content
```

>   - To enhance information browsing

Reviewer 2: ```suggestion
  - To make information discoverable
```

- The user goals that the card-based web design best responds to:
  - scrolling through
  - scanning through
  - browsing through

## When not to use

> - For information searching (as opposed to browsing).

Reviewer 2: ```suggestion
- Searching for information (as opposed to browsing).
```

> - On small screen display avoid using too many cards as they force users to scroll down and rely on their short-term memory. This creates a cognitive overload which harms UX.

Reviewer 2: ```suggestion
- On small screen display avoid using too many cards as they force users to scroll down and rely on their short-term memory. This creates a cognitive overload, a negative user experience.
```

## DO's

- Use cards to emphasize on contents
- Only display the most relevant information on each card
  illustration: card with a thumbnail, title, description
- Keep the amount of links on the cards as low as possible

## DON'Ts

> - Overuse cards; Limit the amount of cards to small groups of item.

Reviewer 2: ```suggestion
- Overuse cards 
- Limit the amount of cards to small groups of items
- Limit to 3-4 cards within a row
```

Author: The suggested change is a nice addition but shouldn't replace the one in place.

Reviewer 2: I replied it.

> - Use cards as an image gallery, use <Link to="/ec/component/media/gallery/usage">gallery</Link> instead.

Reviewer 3: the links are broken (missing "s" to "components")
